src/extension.ts

```diff
@@ -106,27 +106,41 @@ function fetchAndLaunchMetals(context: ExtensionContext, javaHome: string) {
     p => !p.startsWith("-agentlib")
   );
 
+  const customRepositories: string[] = config
+    .get<string>("customRepositories")!
+    .toString()
+    .split(" ")
+    .filter(e => e.length > 0);
+
+  const customRepositoriesCoursierArgumentList = customRepositories.reduce(
+    (acc, elem) => acc.concat(["-r", elem]),
```

Comment:
The Metals server also uses coursier to programmatically download further dependencies like scalafmt, bloop and mtags (Metals completions for non-2.12.8 Scala versions). Can we try using the `COURSIER_REPOSITORIES` environment variable instead?